packages/core/src/fhir-to-cda/cda-templates/components/__tests__/allergy-examples.ts

```diff
@@ -0,0 +1,106 @@
+import { AllergyIntolerance } from "@medplum/fhirtypes";
+
+export const allergyMedication: Partial<AllergyIntolerance> = {
+  category: ["medication"],
+  code: {
+    coding: [
+      {
+        system: "http://www.nlm.nih.gov/research/umls/rxnorm",
+        code: "7980",
+        display: "Penicillin G",
+      },
+    ],
+  },
+  clinicalStatus: {
+    coding: [
+      {
+        system: "http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical",
+        code: "active",
+      },
+    ],
+  },
+  reaction: [
+    {
+      substance: {
+        coding: [
+          {
+            code: "70618",
+            display: "Penicillin",
+            system: "http://www.nlm.nih.gov/research/umls/rxnorm",
+          },
+        ],
+      },
+      severity: "severe",
+      manifestation: [
+        {
+          coding: [
+            {
+              system: "http://snomed.info/sct",
+              code: "247472004",
```

Comment:
why dont we use the manifestation code?

Reply:
use where? for what?

Reply:
Good catch! I missed updating the code for this part